07_safety/01_agency_geo_county.Rmd

```diff
@@ -0,0 +1,731 @@
+
+    Note that we only use the 2022 Batch Header File. Using the Batch Header File from each year might be more accurate, but for the most part it won't matter, especially because so few agencies reported through NIBRS prior to 2021. Additionally, downloading and reading in the Batch Header File for each year takes a lot of time and computing power, and must be done manually for each year which introduces room for error in replication. Future updates may consider using the Batch Header Files for each year, which can be accessed through the following links:
+    * [2014](https://www.icpsr.umich.edu/web/ICPSR/studies/36421)
+    * [2015](https://www.icpsr.umich.edu/web/ICPSR/studies/36851)
+    * [2016](https://www.icpsr.umich.edu/web/ICPSR/studies/37066)
+    * [2017](https://www.icpsr.umich.edu/web/ICPSR/studies/37650)
+    * [2018](https://www.icpsr.umich.edu/web/ICPSR/studies/37649)
+    * [2019](https://www.icpsr.umich.edu/web/ICPSR/studies/38565) 
+    * [2020](https://www.icpsr.umich.edu/web/ICPSR/studies/38566) 
+    * [2021](https://www.icpsr.umich.edu/web/ICPSR/studies/38807)
+    * [2022](https://www.icpsr.umich.edu/web/ICPSR/studies/38925)
+    
+Accessing these data requires an ICPSR account. Manually downloading these data is a five-step process, but you will only need to do each of these steps once for the creation of this metric:
+    1. Create an ICPSR account [here](https://login.icpsr.umich.edu/realms/icpsr/login-actions/registration?client_id=icpsr-web-prod&tab_id=BqotoE6wnqw).
+    2. Navigate to the National Archive of Criminal Justice Data (NACJD) landing page for the year you are accessing (e.g., 2022 is [here](https://www.icpsr.umich.edu/web/NACJD/studies/38925)).
+    3. Select `Download`
+        + Select `Delimited`
+        + Select `Agree` on the `Terms of Use`
+    4. Sign into your ICPSR account
+    5. This will download a zip file titled `ICPSR_38925-V2`. Unzip the file, navigate to `ICPSR_38925\DS0001\38925-0001-Data.tsv`, and move it to the appropriate directory/folder. This is the `Batch Header File`.
+
+
+## 2. Load and clean NIBRS Batch Header File
+This section reads in and cleans the reporting agency data contained in the Batch Header File.
+
+### 2.1 Get universe of agencies from the 2022 NIBRS Batch Header File
+Load 2022 Batch Header File
+```{r, message=FALSE}
+batch_header_file <- read_tsv(here::here("07_safety", "data", "38925-0001-Data.tsv"))
+
+```
+
+Limit file to necessary variables and rename. We use the codebook (`ICPSR_38925\DS0001\38925-0001-Codebook-ICPSR`) to determine what each variable represents.
+```{r}
+# Note that commented out variables were included in the 2021 code for this metric but not used in the analysis. Will delete them at final step.
```

Comment:
Is now the final step? if so, delete

Reply:
Yes. I deleted the commented out variables as well as this note.